scripts/update-config-fixtures.ts

```diff
@@ -0,0 +1,36 @@
+import { readFileSync, writeFileSync } from 'fs';
+import fg from 'fast-glob';
+import { version as packageVersion } from '../packages/airnode-node/package.json';
+
+const configPatterns = ['./packages/**/*config.example.json', './packages/**/*config.valid.json'];
```

Comment:
I found it a bit awkward to use `config.example.json` names for fixtures inside test files. After a few minutes think I settled on using `config.valid.json` pattern. I am open to better alternatives.